tests/test_interrupt_node.py

```
@@ -893,3 +894,776 @@ def decide(messages: list) -> str:
+class TestInterruptNodeLegacyCompat:
+    """Ensure the legacy constructor still works after refactoring."""
+
+    def test_legacy_constructor_unchanged(self):
+        n = InterruptNode(name="approval", input_param="draft", output_param="decision")
+        assert n.name == "approval"
+        assert n.inputs == ("draft",)
+        assert n.outputs == ("decision",)
+        assert n.func is None
+
+    def test_legacy_with_handler(self):
+        handler = lambda x: "yes"
```

Ruff (0.14.14)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `handler` as a `def`

(E731)